crates/aim-downloader/src/io.rs

```diff
@@ -8,13 +8,7 @@ fn get_output_file(path: &str, silent: bool) -> (Option<std::fs::File>, u64) {
             if !silent {
                 println!("File exists. Resuming.");
             }
-            file = Some(
-                std::fs::OpenOptions::new()
-                    .write(true)
-                    .append(true)
-                    .open(path)
-                    .unwrap(),
-            );
+            file = Some(std::fs::OpenOptions::new().append(true).open(path).unwrap());
```

Comment:
any reason to remove `write(true)`?

Reply:
wow...actually this change was made by `autofix-ci[bot]`, I didn't change irrelevant module.

Well, as I check the std doc, [it mentioned](https://doc.rust-lang.org/std/fs/struct.OpenOptions.html#method.append)
```
Note that setting .write(true).append(true) has the same effect as setting only .append(true).
```

I guess that's why the bot removed it.
Perhaps configure the bot to format code ONLY instead of optimizing it?